lib/situm_ar.dart

```diff
@@ -233,8 +235,29 @@ class _ARWidgetState extends State<ARWidget> with WidgetsBindingObserver {
 
   void onUnityViewMessage(UnityViewController? controller, String? message) {
     debugPrint("Situm> AR> MESSAGE! $message");
+
     if (message == "BackButtonTouched") {
       arController.onArGone();
+    } else {
```

Comment:
Eu metería o if para comprobar a mensaxe, aínda que sólo sexa para que sexa máis entendible o código.